tests/test_queries.py

```diff
@@ -29,7 +30,7 @@ class Config:
 def test_get_instance_with_generic_foreign_key():
 
     bookmark = Bookmark.objects.create(url="https://www.djangoproject.com/")
-    Tagged.objects.create(content_object=bookmark, slug="django")
+    tagged = Tagged.objects.create(content_object=bookmark, slug="django")
```

Comment:
Note to my future self: the content type ID incremented after I included the Attachment model at the top of the test app models file, and also a lookup using `ContentType.objects.get_for_model(tagged)` was returning the previously expected value. Probably just because of how the test cases are defined.